CHANGELOG/OWNERS

```diff
@@ -0,0 +1 @@
+# TODO: create alias for release team and decide who is approver and reviewer for release notes
```

Comment:
Thoughts on this? I think we should put the CAPI release team in here but I'm not sure if it should be:
1) as approvers or reviewers or both
2) the whole team or just the leads
3) the comms team only
4) something else?

Reply:
Basically approver rights on this folder allows creating release branches and release tags, right? 

I think giving it to the entire release team is too much. For me this would imply a pretty high bar to include folks in the release team as inclusion in the team would equal those rights (which have been previously only granted to maintainers).

Not sure, but I'm leaning towards only the release lead as approver and the entire time as reviewers. I wouldn't want to raise the bar for becoming CI / comms lead too much.


I think either the release lead or one of the maintainers will always be around.

Reply:
> but I'm leaning towards only the release lead as approver and the entire time as reviewers.

+1 to this

Reply:
added, PTAL